includes/class-wc-product-variable.php

```diff
@@ -503,7 +503,9 @@ protected static function sync_price( &$product ) {
 		if ( ! is_a( $product, 'WC_Product' ) ) {
 			$product = wc_get_product( $product );
 		}
-		$this->data_store->sync_price( $product );
+
+		$data_store = WC_Data_Store::load( 'product_' . $product->get_type() );
```

Comment:
Was `$this->data_store` not already present here from https://github.com/woocommerce/woocommerce/blob/38421fd0953e9987c6daf34dac63655a73c88a0f/includes/abstracts/abstract-wc-product.php#L149 ?